curriculum/structure/superblocks/full-stack-developer.json

```
@@ -179,6 +179,7 @@
             "lecture-working-with-css-flexbox",
             "workshop-flexbox-photo-gallery",
             "lab-page-of-playing-cards",
+            "lab-pricing-plans-layout",
             "review-css-flexbox",
```

```suggestion
            "lab-page-of-playing-cards",
            "review-css-flexbox",
```

we are not adding to fsd anymore